pkg/apis/gloo/gloo/v1/types.go

```diff
@@ -109,11 +110,35 @@ type ConnectionConfig_HttpProtocolOptions struct {
 	HeadersWithUnderscoresAction uint32    `json:"headersWithUnderscoresAction,omitempty"`
 }
 
+type LoadBalancerConfig struct {
+	RoundRobin   *LoadBalancerConfig_RoundRobin   `json:"roundRobin,omitempty"`
+	LeastRequest *LoadBalancerConfig_LeastRequest `json:"leastRequest,omitempty"`
+}
+
+type LoadBalancerConfig_RoundRobin struct {
```

Comment:
```suggestion
type LoadBalancerConfigRoundRobin struct {
```

Reply:
looking at the related PR, these types were snake cased because they were generated from a protobuf but since aren't doing that here, i think we should follow the golang convention and keep it camel cased

Reply:
It did seem like it was generated but I couldn't tell how. Should I be generating it instead? There are technically a lot of other features in Gloo that are not being copied over so it might be good to generate.